Installation/changes.html

```diff
@@ -145,6 +145,24 @@ <h2 id="release4.12">Release 4.12 </h2>
 <!-- Cell Complexes and Polyhedra -->
 <!-- Arrangements -->
 <!-- Triangulations and Delaunay Triangulations -->
+<h3>2D Alpha Shapes</h3>
+<ul>
+  <li> It is now possible to use <code>CGAL::Periodic_2_triangulation_2</code>
+  as underlying triangulation for <code>Alpha_shape_2</code>. </li>
+</ul>
+
+<h3>2D Periodic Triangulations </h3>
+<ul>
+   <li><b>Breaking change</b>:
+   The class <code>Periodic_2_triangulation_hierarchy_vertex_base_2</code>
+   (and its corresponding header) have been removed. Users should directly use the
+   class <code>Triangulation_hierarchy_vertex_base_2</code>, which is identical.</li>
+   <li><b>Breaking change</b>:
+   The functions <code>circumcenter()</code>, <code>side_of_oriented_circle()</code>,
+   and <code>is_extensible_in_1_sheet_h[12]()</code> are related to
+   Delaunay triangulations and have been moved from <code>Periodic_2_triangulation_2</code>
+   to <code>Periodic_2_Delaunay_triangulation_2</code>. </li>
```

Comment:
We usually have compatibility headers containing deprecation warnings. Anything preventing from doing it here?

Reply:
These functions will not compile in P2T2 if you're actually using a model of `Periodic_2TriangulationTraits_2`. I'm not in favor of leaving code compatibility to code that shouldn't work.

Besides, it doesn't break anything for people using P2DT2 (and not a naked P2T2), which I suspect is pretty much all use cases.